lib/form/validation.js

```diff
@@ -69,6 +69,16 @@ const rendered = (type) => {
         };
         inputs[inp.id].siblings = {};
 
+        // check if parent plugin is required
+        if (plugin.hasOwnProperty('required') && plugin.required === 'save') {
```

Comment:
This type of logic seems to be scattered about over a few files. Can you please do this _once_ in `merge`?